src/app/components/shell/footer/footer.js

```diff
@@ -79,6 +78,27 @@ function Footer({
     );
 }
 
+function OsanoToggle() {
+    const osano = typeof window === 'undefined' ? undefined : window.Osano;
+    const showOsanoDrawer = React.useCallback(
+        (e) => {
+            e.preventDefault();
+            osano.cm.showDrawer('osano-cm-dom-info-dialog-open');
+        },
+        [osano]
+    );
+
+    if (osano === undefined || osano.cm.mode === 'debug') {
+        document.body.classList.remove('hide-osano');
+        return null;
+    }
+
+    document.body.classList.add('hide-osano');
+    return (
+        <a href='//' role='button' onClick={showOsanoDrawer}>Manage cookies</a>
```

Comment:
can you use an actual button here?

Reply:
It would require undoing all the styling for button.

Reply:
It looks like Osano brings in its own default styling that includes `top: 0`
![image](https://github.com/openstax/os-webview/assets/7315241/12225472-0ea3-4433-8848-c3292f126d85)
Rex sets bottom, which seems to overrule the top.
```
.osano-cm-dialog--type_bar.osano-cm-dialog--position_bottom {
    bottom: 0;
}
```

Reply:
Position weirdness:
In osweb, the osano-cm-window is position: fixed; bottom: 0; top: auto;
Within it, the dialog is position: relative.

In rex, osano-cm-window is position: absolute; top: 0; height: 0;
Within it, the dialog is position: fixed; bottom: 0.

I can take out the styling for the osano-cm-window and change the dialog to fixed/bottom.

Reply:
Using a button (styled as a button):
![image](https://github.com/openstax/os-webview/assets/7315241/3d935b01-7997-45a3-bc1f-d5e741df7551)

Reply:
re:
> This is due to a sitewide rule setting dialog position.

is this something that we should make more selective? if we changed it would we need all the extra styling?

Reply:
I found that I have an already-defined `.footer-dialog` style; I have piggybacked `.osano-cm-dialog` on it and added a few other lines for a smaller overall osano styling footprint.